.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ commands:
     notify-on-failure:
         steps:
             - slack/notify:
-                  branch_pattern: master, /^\d+\.\d+\.x$/
+                  branch_pattern: master,\d+\.\d+\.x
```

Comment:
sure for this ?

Reply:
yes ! See https://github.com/gravitee-io/gravitee-api-management/commit/1ab21429a4af03364d3746c9d824bc74c534c43b